tools/admin/requests/view-request.ts

```diff
@@ -52,3 +53,22 @@ export type GetApplicationResponse = {
     };
   };
 };
+
+/** Get applicant id from application */
+export const GET_APPLICATION_APPLICANT = gql`
+  query GetApplicationProcessing($id: Int!) {
+    application(id: $id) {
+      applicant {
+        id
+      }
+    }
+  }
+`;
+
```

Comment:
This query is not necessary. One of the main benefits of GraphQL is that you can query for whatever you need in just one query. Thus, you can select `applicant > id` from the application in the `GET_APPLICATION_QUERY` above.